16x9/DialogFullScreenInfo.xml

```diff
@@ -5,6 +5,7 @@
 	<onunload>Skin.Reset(PausedOverlayCast)</onunload>
 	<onunload>Skin.Reset(ShowPausedOverlayFlags)</onunload>
 	<onunload>Skin.Reset(ShowSeekbar)</onunload>
+	<onunload condition="VideoPlayer.Content(livetv) + !Skin.HasSetting(ToggleLiveTVNext)">Skin.ToggleSetting(ToggleLiveTVNext)</onunload>
```

Comment:
That doesn't work:
<onunload>Skin.Reset(ToogleLiveTVNext)</onunload>
even if I switch the visible-condition with "!" in the NOW and NEXT grouplists. Is this the culprit?
<visible>Skin.HasSetting(ToggleLiveTVNext)</visible>